libs/core/execution/include/hpx/execution/traits/detail/simd/vector_pack_get_set.hpp

```diff
@@ -24,7 +27,11 @@ namespace hpx { namespace parallel { namespace traits {
     HPX_HOST_DEVICE HPX_FORCEINLINE auto set(
         Vector& vec, std::size_t index, T val)
     {
+#if defined(HPX_HAVE_DATAPAR_STD_EXPERIMENTAL_SIMD)
         vec[index] = val;
+#elif defined(HPX_HAVE_DATAPAR_SVE)
+        vec.set(index, val);
+#endif
```

Comment:
This distinction is not needed anymore as you have implemented `datapar::set` in your namespace wrapper.